KsApi/models/templates/ProjectTemplates.swift

```diff
@@ -15,7 +15,7 @@ extension Project {
     blurb: "A fun project.",
     category: .template,
     country: .us,
-    creator: User.template |> \.stats.createdProjectsCount .~ 1,
+    creator: User.template |> \.stats.createdProjectsCount .~ 2,
```

Comment:
Could you explain why you updated the template?

Reply:
This was changed so that I could check `isRepeatCreator` to be true. That property requires that `createdProjectsCount` be greater than 1